### PR TITLE
Stock quantity_allocated on field level

### PR DESCRIPTION
**Pull request by tomaszszymanski129**
I want to merge this change because it's a 3.1 port of https://github.com/saleor/saleor/pull/8851

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5062 Denormalize quantity allocated](https://app.clickup.com/t/2549495/SALEOR-5062)

File: saleor/settings.py [hunk 1]
```diff
@@ -534,6 +534,10 @@ def get_host():
         "task": "saleor.giftcard.tasks.deactivate_expired_cards_task",
         "schedule": crontab(hour=0, minute=0),
     },
+    "update-stocks-quantity-allocated": {
+        "task": "saleor.warehouse.tasks.update_stocks_quantity_allocated_task",
+        "schedule": timedelta(days=1),
```
Comment (L3str4nge) [inline saleor/settings.py:539]: I would change it to `crontab` for some night hour when potentially there is smaller traffic.

**Review by L3str4nge — APPROVED** [review]

File: saleor/settings.py (hunk 1 above)
Comment (IKarbowiak) [reply saleor/settings.py:539]: Why do we need this at all? Migration isn't enough?

Comment (tomaszszymanski129) [reply saleor/settings.py]: We want to keep it to check for any inconsistency in logs for a while, however running at night is a good idea, changing to crontab.

**Review by fowczarek — APPROVED** [review]

**Pull request merged**